cli/config.go

```diff
@@ -0,0 +1,83 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/mainflux/mainflux/pkg/errors"
+	"github.com/pelletier/go-toml"
+)
+
+type Config struct {
+	AuthToken string `toml:"auth_token"`
+}
+
+// Save - store config in a file
+func Save(c Config, file string) error {
+	b, err := toml.Marshal(c)
+	if err != nil {
+		return errors.New(fmt.Sprintf("failed to read config file: %s", err))
+	}
+	if err := ioutil.WriteFile(file, b, 0644); err != nil {
+		return errors.New(fmt.Sprintf("failed to write config TOML: %s", err))
+	}
+	return nil
+}
+
+// Read - retrieve config from a file
+func Read(file string) (Config, error) {
```

Comment:
Are `Read` and `GetConfigPath` used by any other package? If not, there is no need to export them.

Reply:
If you decide to make them not exported, please move the unexported code after the exported methods (`Save` and `Parse`).

Reply:
Makes sense, updated.